src/components/VaultItemPopOut.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+  Stack,
+  TextField,
+} from "@mui/material";
+
+export default function VaultItemPopOut({ vaultItem, open, closePopup }) {
+  const [showPassword, setShowPassword] = useState(false);
+
+  return (
+    <Dialog open={open} fullWidth maxWidth="sm">
+      <DialogTitle>Account Details</DialogTitle>
+      <DialogContent>
+        <Stack spacing={2} margin={2}>
+          <TextField
+            defaultValue={vaultItem.name}
+            variant="outlined"
+            label="Account Name"
+            inputProps={{ readOnly: true }}
+          ></TextField>
+          <TextField
+            defaultValue={vaultItem.url}
+            variant="outlined"
+            label="URL"
+            inputProps={{ readOnly: true }}
+          ></TextField>
+          <TextField
+            defaultValue={vaultItem.username}
+            variant="outlined"
+            label="Username"
+            inputProps={{ readOnly: true }}
+          ></TextField>
+          <TextField
+            defaultValue={vaultItem.password}
+            variant="outlined"
+            label="Password"
+            type={showPassword ? "text" : "password"}
+            inputProps={{ readOnly: true }}
+            onClick={() => setShowPassword(!showPassword)}
+          ></TextField>
```

Comment:
Just realized collection name isn't on vault item. We might have to search the collections list with the vault_collection uuid to get the collection name or come up with something else. I would create a separate task for that.